### DISCUSSION TITLE
About this project

### DISCUSSION THREAD

**Discussion by MadhushaPrasad** [MadhushaPrasad/Diagon-Alley#8] [Announcements]
This is an open library for every tech student to level up their tech knowledge by reading e-books.
You can read any type of e-book from here by searching it from the category list also there needs to be only a read option we cant add a download option here.
we will discuss the download option later after releasing our first version.
And Also need to add a feedback section to get new ideas from users.

**Comment by Dhanuka99** [comment]
I interested in database section.. how to expecting create database and which technologies are use our project
